session.go

```diff
@@ -366,15 +383,20 @@ func (s *session) EnqueueBytesAndSend(msg []byte) {
 	s.sendQueued()
 }
 
-func (s *session) sendBytes(msg []byte) {
+func (s *session) sendBytes(msg []byte) bool {
```

Comment:
minor: for a function named `sendBytes` returning a bool, I would assume it returns whether it successfully sent and not the opposite. I understand it's the opposite because you named the var `blocked` but would personally change it to `sent`

Reply:
Addressed: https://github.com/alpacahq/quickfix/pull/18/commits/21384e6dc24b832a5b3613f534b1256d6d4ebf85